.github/workflows/test-suite.yml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: "${{ matrix.os }}"
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10.0-beta.2"]
```

Released 2 hours ago :)
```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10.0-beta.3"]
```